src/binpickingtask.cpp

```diff
@@ -153,12 +155,15 @@ void BinPickingTaskResource::Initialize(const std::string& defaultTaskParameters
 }
 #endif
 
-boost::property_tree::ptree BinPickingResultResource::GetResultPtree() const
+void BinPickingResultResource::GetResultJson(rapidjson::Document& pt) const
 {
     GETCONTROLLERIMPL();
-    boost::property_tree::ptree pt;
+    //rapidjson::Document d(rapidjson::kObjectType);
     controller->CallGet(boost::str(boost::format("%s/%s/?format=json&limit=1")%GetResourceName()%GetPrimaryKey()), pt);
```

Comment:
Sorry for asking, but here, is passing rapidjson::Document directly safe?
I mean, if the json could contain null, we have to add a workaround in mujinjson.h.

Reply:
ah already done in https://github.com/mujin/controllerclientcpp/commit/41501ab ... thanks @akovski